src/stack.ts

```diff
@@ -51,7 +51,7 @@ export function stack(mark: Mark, encoding: Encoding, stacked: StackOffset): Sta
 
   // Should have grouping level of detail
   const stackByChannels = STACK_GROUP_CHANNELS.reduce((sc, channel) => {
-    if (has(encoding, channel) && !encoding[channel].aggregate) {
+    if (has(encoding, channel) && !('aggregate' in encoding[channel])) {
```

Comment:
Error TS2339: Property 'aggregate' does not exist on type 'FieldDef | ChannelDefWithLegend | OrderChannelDef | FieldDef[] | OrderChannelDef[]'.
  Property 'aggregate' does not exist on type 'FieldDef[]'.